src/common/snippets/tests/src/lowering_utils.cpp

```diff
@@ -107,7 +109,6 @@ std::shared_ptr<ov::snippets::op::Subgraph> LoweringTests::getSubgraph(const std
 
 std::shared_ptr<ov::snippets::op::Subgraph>
         LoweringTests::getLoweredSubgraph(const std::shared_ptr<Model> &f,
-                                          const ov::PartialShape& master_shape,
```

Comment:
Could you tell me please what is the reason? Might be I missed something

Reply:
I noticed that `master_shape` is not used anywhere. So I decided to remove this useless outdated parameter in this PR